src/util/addresses.ts

```diff
@@ -145,6 +152,13 @@ export const WETH9: {
     'WETH',
     'Wrapped Ether'
   ),
+  [ChainId.SEPOLIA]: new Token(
+    ChainId.SEPOLIA,
+    '0xD0dF82dE051244f04BfF3A8bB1f62E1cD39eED92',
```

Comment:
this is the WETH address that we (Uniswap) deployed: `0xfFf9976782d46CC05630D1f6eBAb18b2324d6B14`